lib/src/util/windows/process.cc

```diff
@@ -0,0 +1,31 @@
+#include <pxp-agent/util/process.hpp>
+
+#include <leatherman/windows/windows.hpp>
+#include <leatherman/windows/system_error.hpp>
+
+#define LEATHERMAN_LOGGING_NAMESPACE "puppetlabs.pxp_agent.util.windows.process"
+#include <leatherman/logging/logging.hpp>
+
+namespace PXPAgent {
+namespace Util {
+
+namespace lth_win = leatherman::windows;
+
+bool processExists(int pid) {
```

Comment:
Will do.
I didn't put that in leatherman in the first place because, as you know, we need to speed up this.